systemd/coreos-installer-noreboot.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Give Login Shell After CoreOS Installer
+After=coreos-installer.service
+ConditionPathExists=!/run/coreos-installer-reboot
+
+[Service]
+Type=oneshot
+StandardInput=tty-force
+StandardOutput=inherit
+StandardError=inherit
+ExecStartPre=/usr/bin/echo -e "\nCoreOS install complete. Starting login shell\n"
+ExecStart=/usr/sbin/sulogin --force
```

Comment:
Ahh, interesting. I think this overlaps with my previous question re. the reboot service.

So I guess the question is: do we want this boot to be special or just like any other boot (but with code that runs the installer in the background). IOW, if users want to use a shell after installation, they could just provision the system as usual via Ignition and login, right?

I guess, I'm trying to wrap my head around whether we want to do this, when IIUC part of the idea was to move towards `coreos-installer` just be a regular binary you can run from a normal FCOS/RHCOS boot (or script to run via Ignition), and leave the kargs approach as a "legacy" thing.

Reply:
> and leave the kargs approach as a "legacy" thing.

Though I notice Benjamin's patch adds more kargs to the list for e.g. FCOS streams, so maybe I'm mistaken here.

And OK yeah, I totally missed that we're still overriding `default.target` here. So this WFM! (I thought we were just running this in the background of an otherwise normal boot.)

Reply:
OK, I've split out some thoughts on this in https://github.com/coreos/coreos-installer/issues/124. But anyway, it doesn't have to block on this, though... can we not add new kargs for the time being?

Reply:
>  can we not add new kargs for the time being?

it looks like the ones he added were `coreos.inst.insecure` `coreos.inst.stream` `coreos.inst.stream_base_url`. Those were mostly convenience based on the new functionality within the rust installer. I'd prefer not to rip them out, but also want to get this merged ASAP, so I'll go either way.

Reply:
Note that the tool defaults to the stable stream. If we don't provide any way to change the stream then anyone trying the installer today (when no stable stream exists) and not providing their own disk image url will experience a failure.

Reply:
and I think we need coreos.inst.insecure too because otherwise you can't even locally test the kargs approach right now without it.

Reply:
Hmm, at least `coreos.inst.stream_base_url`, it seems like we can drop, right? If you're targeting your own image, you can just use `coreos.inst.image_url` for now.

After that, yeah let me finish testing and then get this in!

Reply:
yes, that one options seems to be more useful for us testing out the other parts of the metadata we publish. If you'd like I can try to drop that option.